.github/workflows/dotnet.yml

```diff
@@ -118,6 +118,7 @@ jobs:
           echo "sentry_dsn=\"${{secrets.SENTRY_DSN}}\"" >> env.tfvars
           echo "sor_configuration_production=\"${{secrets.SOR_CONFIG_PRODUCTION}}\"" >> env.tfvars
           echo "sor_configuration_staging=\"${{secrets.SOR_CONFIG_STAGING}}\"" >> env.tfvars
+          echo "service_plan_name=\"${{secrets.SERVICE_PLAN_NAME}}\"" >> env.tfvars
```

Comment:
To check my understanding, we're removing the service plan resource and replacing it with references to a stored environment value instead?

Reply:
yes, we are removing the resource because it was already created manually as all the 3 apis live under the same app service plan. therefore we can just reference it using a data block as it already exists.